assets/libraries/cloudformation/library.rego

```diff
@@ -1,5 +1,52 @@
 package generic.cloudformation
 
+hasSecretManager(str, document) {
+	selectedSecret := strings.replace_n({"${": "", "}": ""}, regex.find_n(`\${\w+}`, str, 1)[0])
+	document[selectedSecret].Type == "AWS::SecretsManager::Secret"
+}
+
+# Check if the type is ELB
+isLoadBalancer(resource) {
+	resource.Type == "AWS::ElasticLoadBalancing::LoadBalancer"
+}
+
+# Check if the type is ELB
+isLoadBalancer(resource) {
+	resource.Type == "AWS::ElasticLoadBalancingV2::LoadBalancer"
+}
+
+# Check if array contains an element
+arrayContains(elem, array) {
+	array[_] == elem
+}
```

Comment:
the function **inArray** of common library achieves the same as this right? If so, we can use the inArray instead and delete this